frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -132,7 +132,7 @@ const deleteAppAdmin = async (admin: FamAppAdminGetResponse) => {
 
 <template>
     <ManagePermissionsTitle :isApplicationSelected="isApplicationSelected" />
-
+    {{ LoginUserState.state.value.famLoginUser?.accesses }}
```

Comment:
Hi Nick, just wondering maybe your forgot to remove this line?

Reply:
I removed it.

Reply:
Sorry about that. This shouldn't be committed indeed, I think after merging with main, git added it anyway unnoticed. Sorry about the trouble and thanks for removing it.